source/_components/light.blinkt.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Blinkt!"
```

Comment:
@Landrash The component's original name actually includes the `!` (see https://shop.pimoroni.com/products/blinkt), of course I'm happy to remove it, just wondering if there's a specific reason to do so? (does it mess with these docs some way?)